### PR TITLE
[doc]unify format

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Changes in this pr:
1.  Unify query format in doc to `nebula> …`
2. Add insert multiple vertices syntax.

### CODE REVIEW

**Comment by Amber1990Zhang** [comment]
> In `ngql` code blocks, I think the `nebula>` prefix is not necessary!

Thanks for your suggestion. But I want to make the doc more clear. @whitewum

**Comment by yixinglu** [comment]
> > In `ngql` code blocks, I think the `nebula>` prefix is not necessary!
> 
> Thanks for your suggestion. But I want to make the doc more clear. @whitewum

ngql code blocks are not bash commands and some multiple query statements concatenated with `\` also needn't the prefix.

**Comment by Amber1990Zhang** [comment]
> > > In `ngql` code blocks, I think the `nebula>` prefix is not necessary!
> > 
> > 
> > Thanks for your suggestion. But I want to make the doc more clear. @whitewum
> 
> ngql code blocks are not bash commands and some multiple query statements concatenated with `\` also needn't the prefix.

Yes, there is no need to add the prefix when using `\`

File: docs/manual-CN/2.query-language/2.functions-and-operators/set-operations.md [hunk 1]
```diff
@@ -15,17 +15,17 @@
 ### 示例
 
 ```ngql
-GO FROM 1 OVER e1 \
+nebula> GO FROM 1 OVER e1 \
 UNION \
-GO FROM 2 OVER e1
+nebula> GO FROM 2 OVER e1
```
Comment (whitewum) [inline docs/manual-CN/2.query-language/2.functions-and-operators/set-operations.md]: 错了

File: docs/manual-CN/2.query-language/2.functions-and-operators/set-operations.md [hunk 2]
```diff
@@ -15,17 +15,17 @@
...
 ```
 
 以上语句返回点 `1` 和 `2` (沿边 `e1`) 关联的唯一的点。
 
 ```ngql
-GO FROM 1 OVER e1 \
+nebula> GO FROM 1 OVER e1 \
 UNION ALL\
-GO FROM 2 OVER e1
+nebula> GO FROM 2 OVER e1
```
Comment (whitewum) [inline docs/manual-CN/2.query-language/2.functions-and-operators/set-operations.md]: 错了

File: docs/manual-CN/2.query-language/2.functions-and-operators/set-operations.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # 集合操作 (`UNION`，`INTERSECT`， `MINUS`)
```
Comment (wilsonyou) [inline docs/manual-CN/2.query-language/2.functions-and-operators/set-operations.md]: please use comma in the brackets.

Comment (wilsonyou) [inline docs/manual-CN/2.query-language/2.functions-and-operators/set-operations.md]: ，-> 、

File: docs/manual-EN/2.query-language/3.language-structure/property-reference.md [hunk 4]
```diff
@@ -20,49 +20,49 @@ and `prop_name` specifies the property name.
...
 
 ```ngql
-GO FROM 1 OVER e1 YIELD $^.start.name AS startName, $$.end.Age AS endAge
+nebula> GO FROM 1 OVER e1 YIELD $^.start.name AS startName, $$.end.Age AS endAge
 ```
 
-to get the starting vertex's property name and ending vertex's property age.
+Use the above query to get the source vertex's property name and ending vertex's property age.
 
 ## Reference from edge
```
Comment (wilsonyou) [inline docs/manual-EN/2.query-language/3.language-structure/property-reference.md]: uppercase

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md [hunk 5]
```diff
@@ -85,16 +85,16 @@ GO FROM <node_list> OVER <edge_type_list | *> YIELD | YIELDS [DISTINCT] <return_
 For example:
 
 ```ngql
-GO OVER edge1, edge2....  // traverse alone edge1 and edge2 or
-GO OVER *   // * means traverse along all edge types
+nebula> GO OVER edge1, edge2....  // traverse alone edge1 and edge2 or
+nebula> GO OVER *   // * means traverse along all edge types
```
Comment (jievince) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md]: > GO OVER *

is wrong. Should be GO FROM 101 OVER *;

**Pull request merged**